### PR TITLE
feature/503

### LINKED ISSUE #503: update travis messaging
https://docs.travis-ci.com/user/notifications/#Configuring-slack-notifications

```yaml
notifications:
  slack:
    template:
      - "Build %{result} in %{duration} - %{branch} by %{author}"
      - "<%{build_url}|#%{build_number}> (<%{compare_url}|%{commit}>)"
      - "%{commit_message}"
```

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #503.

Customizes Travis CI build messages in slack #dev channel.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,11 @@ cache:
...
+    rooms:
+      - secure: "Xti6UQw2kAqVJ6STI1I1VFadPv6Z+7KRHk+7UF9eVtiXks2fkKtl07svpMh81wTkRRMcbpCUb7i8AujcuFOBBMmv4A0HnMM8oOOH0hTsxacs5waYlfvDfXhliJKYjuTIHg55li5VqJYoZsQqJ9gSuegE5f9oQ3fmUAgfDXkwChNoinSEs4IIhMkGYV77PniYRdL4Sj92yQhlhu4YTz4EbOoAggdLAp11m8ALpnxShDAJUsPJQum0ZtYBr+yTacdQIAjncZknuhNkeJvblZkUDDun57uL/Wv6uJlfEBxclD+FD4YOKSoiXla5N5IMhzwJIO8umd2CGy109frZO04Fcwq/fEXcj0N38mOkCluP8IXx3b7G8DdMD9MMkv2Mac/EC9gTIB7zcMmugBigGB7kFhttdmDGvTsjAKVugfWjSu+AUXRHxmR4uohGKaUi1G6gKdOZDITh1gRi5j0S+yypE15uRypMgjex4ZqfcydPrc3pAmAS3j2YTVbIfbBP4jUMj3Av2bNdEh3moFcGh+2O/FGsOmzHYBw1YNhu5b+FesFShtxoKKSJi/DaLd9nODrbPU0gbJ+xBuDDMPUMwNwqEk2aMd7hUbLqST8ss237u816aG4KiK9Ipgp+hqcKHYne20dQYt7SX0KV8suoxcYqgcs6/igl8cPtgUrgzOE21i4="
+    template:
+      - Build %{result} - %{author} on <https://www.github.com/openscope/openscope/%{branch}|%{branch}>
+      - <%{build_url}|#%{build_number}> (<%{compare_url}|%{commit}>) %{result} in %{duration}
+      - "%{commit_subject}"
```
Comment (n8rzz) [inline .travis.yml]: should this one be `commit_message` instead?

File: .travis.yml [hunk 2]
```diff
@@ -15,6 +15,11 @@ cache:
...
+    rooms:
+      - secure: "Xti6UQw2kAqVJ6STI1I1VFadPv6Z+7KRHk+7UF9eVtiXks2fkKtl07svpMh81wTkRRMcbpCUb7i8AujcuFOBBMmv4A0HnMM8oOOH0hTsxacs5waYlfvDfXhliJKYjuTIHg55li5VqJYoZsQqJ9gSuegE5f9oQ3fmUAgfDXkwChNoinSEs4IIhMkGYV77PniYRdL4Sj92yQhlhu4YTz4EbOoAggdLAp11m8ALpnxShDAJUsPJQum0ZtYBr+yTacdQIAjncZknuhNkeJvblZkUDDun57uL/Wv6uJlfEBxclD+FD4YOKSoiXla5N5IMhzwJIO8umd2CGy109frZO04Fcwq/fEXcj0N38mOkCluP8IXx3b7G8DdMD9MMkv2Mac/EC9gTIB7zcMmugBigGB7kFhttdmDGvTsjAKVugfWjSu+AUXRHxmR4uohGKaUi1G6gKdOZDITh1gRi5j0S+yypE15uRypMgjex4ZqfcydPrc3pAmAS3j2YTVbIfbBP4jUMj3Av2bNdEh3moFcGh+2O/FGsOmzHYBw1YNhu5b+FesFShtxoKKSJi/DaLd9nODrbPU0gbJ+xBuDDMPUMwNwqEk2aMd7hUbLqST8ss237u816aG4KiK9Ipgp+hqcKHYne20dQYt7SX0KV8suoxcYqgcs6/igl8cPtgUrgzOE21i4="
+    template:
+      - Build %{result} - %{author} on <https://www.github.com/openscope/openscope/%{branch}|%{branch}>
```
Comment (n8rzz) [inline .travis.yml]: I'm on the fence with the last part of this line:
```yaml
# option one
%{author} on <https://www.github.com/openscope/openscope/%{branch}|%{branch}>

# option two
<https://www.github.com/openscope/openscope/%{branch}|%{branch}> by %{author}
```

File: .travis.yml (hunk 1 above)
Comment (erikquinn) [inline .travis.yml]: If you want the whole multiline message included in the slack message. I thought the first line would be sufficient, but am 100% cool with showing the whole thing if you want to

File: .travis.yml (hunk 2 above)
Comment (erikquinn) [inline .travis.yml]: I definitely prefer the first convention over the second, so if you're on the fence, I'd like to stick with that one.

Comment (n8rzz) [inline .travis.yml]: Works for me!

File: .travis.yml (hunk 1 above)
Comment (n8rzz) [inline .travis.yml]: Good point.  I suppose if that something I _need_ to see I can click one of the links and find it.  Lets keep it as is.  We can always change it later if we don't like it.

**Pull request merged**